src/TLuaInterpreter.cpp

```diff
@@ -3203,94 +3203,178 @@ int TLuaInterpreter::hideUserWindow(lua_State* L)
     return 0;
 }
 
+// No documentation available in wiki - internal function
+void TLuaInterpreter::setBorderSize(lua_State* L, int size, int position, bool updateEvent)
+{
+    Host& host = getHostFromLua(L);
+    // position: 0 = top, 1 = right, 2 = bottom, 3 = left
+    switch(position) {
+        case 0: host.mBorderTopHeight = size; break;
+        case 1: host.mBorderRightWidth = size; break;
+        case 2: host.mBorderBottomHeight = size; break;
+        case 3: host.mBorderLeftWidth = size; break;
+    }
+    if (updateEvent) {
+        int x, y;
+        x = host.mpConsole->width();
+        y = host.mpConsole->height();
+        QSize s = QSize(x, y);
+        QResizeEvent event(s, s);
+        QApplication::sendEvent(host.mpConsole, &event);
+    }
+}
+
+// Documentation: https://wiki.mudlet.org/w/Manual:Lua_Functions#setBorderSizes
+int TLuaInterpreter::setBorderSizes(lua_State* L)
+{
+    int sizeTop;
+    if (!lua_isnumber(L, 1)) {
+        lua_pushfstring(L, "setBorderSizes: bad argument #1 value (new top size as number expected, got %s!)", luaL_typename(L, 1));
+        lua_error(L);
```

Comment:
Remember the shorter one line equivalent of this and the next one that avoids linters getting confused about the **unreachable** `return`:
```
return lua_error(L);
```

Reply:
Removed that from my code, but found ~320 results in TLuaInterpreter.cpp alone for `(\s*)lua_error\(L\);\n(\s*)return 1;` ... Fix it in which PR? 🤔

Reply:
Definitely the next :)

Reply:
Yeah - it is an ongoing WIP to clean them out - but it helps if any new instances are in the better form from the off... :stuck_out_tongue_winking_eye: